chainer/training/trainer.py

```diff
@@ -184,7 +185,10 @@ def extend(self, extension, name=None, trigger=None, priority=None,
             trigger (tuple or Trigger): Trigger object that determines when to
                 invoke the extension. If it is ``None``, ``extension.trigger``
                 is used instead. If the trigger is not callable, it is passed
-                to :class:`IntervalTrigger` to build an interval trigger.
+                to :class:`IntervalTrigger` to build an interval trigger. If
```

Comment:
Yes, that's what I also though a bit when I fixed originally. Changed the order as suggested.